tracing/src/main/java/com/palantir/remoting3/tracing/AsyncSpanObserver.java

```diff
@@ -75,7 +76,7 @@ public void onFailure(Throwable error) {
             });
         } else {
             log.trace("Failed to notify span observer since the maximum number of allowed concurrent observations was "
-                    + "exceeded: {}", maxInflights);
+                    + "exceeded: {}", SafeArg.of("maxInflights", maxInflights));
```

Comment:
Log message should get cleaned up to remove braces.

Reply:
why? it'll log the safe arg there

Reply:
per the spec, the substitution never happens/the args are named and shown along side.